src/streams/EditStreamCard.js

```diff
@@ -5,45 +5,96 @@ import { View } from 'react-native';
 
 import Input from '../common/Input';
 import ZulipTextIntl from '../common/ZulipTextIntl';
-import SwitchRow from '../common/SwitchRow';
 import ZulipButton from '../common/ZulipButton';
-import styles, { createStyleSheet } from '../styles';
-import { IconPrivate } from '../common/Icons';
+import styles from '../styles';
+import SelectableOptionRow from '../common/SelectableOptionRow';
 
-const componentStyles = createStyleSheet({
-  switchRow: {
-    paddingLeft: 8,
-    paddingRight: 8,
-  },
-});
+type AccessPolicy =
+  | 'public'
+  | 'web_public'
+  | 'private_shared_history'
+  | 'private_protected_history';
```

Comment:
Can simplify this a bit and reduce redundancy:
```suggestion
type AccessPolicy = $Keys<typeof streamAccessPolicies>;
```

(Then either move it after `streamAccessPolicies`, or just say `// eslint-disable-next-line no-use-before-define`.)